libs/vm-modules/src/math/tensor/tensor.cpp

```diff
@@ -57,6 +57,13 @@ VMTensor::VMTensor(VM *vm, TypeId type_id)
   , estimator_(*this)
 {}
 
+VMTensor::VMTensor(VM *vm, TypeId type_id, std::string const &str)
+  : Object(vm, type_id)
+  , estimator_(*this)
+{
+  tensor_ = tensor_.FromString(str);
```

Comment:
FromString is a static method, so I guess it could be moved to initializer list as

VMTensor::VMTensor(VM *vm, TypeId type_id, std::string const &str)
  : Object(vm, type_id)
  , estimator_(*this)
  , tensor_(Tensor<DataType>::FromString(str))
{
}